src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -27,4 +22,29 @@ public Integer getLength() {
     public String getColor() {
         return color;
     }
+
+    @Override
+    public boolean equals(Object object) {
+        if (this == object) {
+            return true;
+        }
+        if (object == null || getClass() != object.getClass()) {
+            return false;
+        }
+        Rectangle rectangle = (Rectangle) object;
+        return (length == null || rectangle.length == null)
```

Comment:
```suggestion
        return length == null
```this condition should be enough